### PR TITLE
Add --sync CLI option to update subcommand

### PR DESCRIPTION
**Pull request by Cypher1**
Adds `--sync` to the `update` subcommand to match `install`.

### CODE REVIEW

**Review by radoering — CHANGES_REQUESTED** [review]
Makes sense. Can you update the docs and add a test analogous to https://github.com/python-poetry/poetry/blob/776fcb8ae3ea76a9e7681054034613ca440c7e9a/tests/console/commands/test_install.py#L158-L169, please?

**Comment by Cypher1** [comment]
Sounds good. Will get to it soon. Thx

**Comment by Cypher1** [comment]
pre-commit.ci autofix

File: tests/console/commands/test_update.py [hunk 1]
```diff
@@ -80,3 +83,21 @@ def test_update_prints_operations(
 
     assert ("Package operations:" in output) is expected
     assert ("Installing docker (4.3.1)" in output) is expected
+
+
+@pytest.fixture()
```
Comment (radoering) [inline tests/console/commands/test_update.py:88]:
```suggestion
```

not a fixture but a test

**Review by radoering — CHANGES_REQUESTED** [review]

File: tests/console/commands/test_update.py (hunk 1 above)
Comment (Cypher1) [reply tests/console/commands/test_update.py]: Ah. Sorry, misunderstood, thought this was injecting the fixtures. New to pytest.

**Comment by Cypher1** [comment]
pre-commit.ci autofix

**Review by radoering — APPROVED** [review]

**Comment by Cypher1** [comment]
Thanks!

**Pull request merged**